CRM/Event/Cart/Form/Checkout/Payment.php

```diff
@@ -744,7 +744,7 @@ public function setDefaultValues() {
    * @param array &$price_set_amount
    * @param &$cost
    */
-  protected function apply_discount($discountCode, &$price_set_amount, &$cost) {
+  protected function apply_discount($discountCode, &$price_set_amount, &$cost, &$event_id) {
```

Comment:
this &$event_id as opposed to $event_id is a bad awful pattern that is used so much in the CiviCRM codebase it's easy to mistake it for the encouraged pattern (but it won't be the cause of the jenkins style issues failure)